template.yaml

```diff
@@ -155,7 +155,7 @@ Resources:
           Type: Api
           Properties:
             RestApiId: !Ref NvaCristinProxyApi
-            Path: /project/{id}
+            Path: /project/{identifier}
```

Comment:
Will fetching one project still work if you change variable name?

Reply:
It's supposed to be renamed all the way- also when getting parameter from RequestInfo